init.sh.tpl

```diff
@@ -3,3 +3,4 @@
 echo "SRE_TOOLING_SERVER_GROUP=\"${group}\"" >> /etc/default/sre-tooling
 echo "SRE_TOOLING_REGION=\"${region}\"" >> /etc/default/sre-tooling
 
+/opt/mybatis/mybatis-migrations-${mybatis_version}/bin/migrate up --path=/tmp/opensrp/configs/assets/migrations --env=deployment
```

Comment:
Is this always required? Can it be optional at some point? I think for a shared module that may be necessary.